andela/wg-ravens
# #157589656 Allow Managers to Delete Trainers From a Gym 157589656

#### What does this PR do?
Enables gym managers to delete or deactivate trainers from a gym

#### Description of Task to be completed?
Gym managers should be able to delete or deactivate trainers from a gym

#### How should this be manually tested?
Login to wger as a gym manager and select a gym to list members. Click on a Gym member with a label trainer, click actions dropdown and choose delete and the user should be deleted

#### What are the relevant pivotal tracker stories?
#157589639
